cmd/security-bootstrapper/res-bootstrap-redis/configuration.toml

```
@@ -39,3 +39,6 @@ TokenFile = '/vault/config/assets/resp-init.json'
   Timeout = 5000
   Type = 'redisdb'
 
+[DatabaseConfig]
+  Path = '/user/local/etc/redis/conf'
```

```suggestion
  Path = '/usr/local/etc'
```
or
```suggestion
  Path = '/usr/local/etc/redis'
```

??

Is there a mkdir somewhere to make sure this directory exists?

And a path.join so that Path doesn't need to end in a slash?

---

yea, it is in the handlers of go code.